src/datachannel.cc

```diff
@@ -258,7 +258,6 @@ NAN_METHOD(DataChannel::Send) {
       arraybuffer = view->Buffer();
     }
 
-    v8::ArrayBuffer::Contents content = arraybuffer->Externalize();
     rtc::Buffer buffer(content.Data(), content.ByteLength());
```

Comment:
This line fails because we no longer define `content`. I am not sure how to get a hold of the `v8::ArrayBuffer::Contents` or the underlying data without `Externalize`—I need to read up on ArrayBuffer